client/flutter/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: WHO Flutter app
 version: 0.1.0+1
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.3.0 <3.0.0"
```

Comment:
I'm not familiar enough with Dart sdk versions.  What impact does this have on device support (if any)?

Reply:
None.  This is to allow use of newer language features.

Reply:
You should be able to bump it up to 2.7.0 if you run the latest flutter version on the stable channel.

Reply:
The hotfix containing that was just recently release, but yes that should be possible.  It isn't needed for this patch specifically though.